src/components/screens/editor/draft-editor-screen.tsx

```diff
@@ -206,6 +209,64 @@ const DraftEditorScreen: React.FC = () => {
     />
   );
 
+  /**
+   * Saves draft form as a template
+   *
+   * @param templateTitle template title string
+   */
+  const saveTemplate = async (templateTitle: string) => {
```

Comment:
It seems that user can save template with exactly the same name/title as already exists. This might become a problem so let's add a check for the template title so it is unique. This check should also be in the backend but that is future development.